### PR TITLE
Fix deprecated rclcpp::Duration() calls

### PR DESCRIPTION
**Pull request by SyllogismRXS**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | (add tickets here #1) |
| Primary OS tested on | (Ubuntu) |
| Robotic platform tested on | (gazebo simulation of Clearpath Dingo) |

---

## Description of contribution in a few bullet points

* The latest ros2 API on master deprecated the Duration(rcl_duration_value_t nanoseconds) call without using std::chrono::nanoseconds or rclcpp::Duration::from_nanoseconds(): https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/duration.hpp#L42 . This commit fixes the deprecated Duration calls.

## Description of documentation updates required from your changes
none

---

## Future work that may be required in bullet points
none

This fix doesn't make sense for the foxy release, but should be included for the next release of ros2. Should the slam_toolbox have a "ros2" branch to incorporate the latest API changes from the ros2 master branch?

### CODE REVIEW

File: include/slam_toolbox/visualization_utils.hpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ inline visualization_msgs::msg::Marker toMarker(
   marker.color.b = 0.0;
   marker.color.a = 1.;
   marker.action = visualization_msgs::msg::Marker::ADD;
-  marker.lifetime = rclcpp::Duration(0.);
+  marker.lifetime = rclcpp::Duration::from_nanoseconds(0);
```
Comment (SteveMacenski) [inline include/slam_toolbox/visualization_utils.hpp:49]: from_nanoseconds doesn't exist in Foxy, that's a master build addition, we need to retarget this to a ros2 branch

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: include/slam_toolbox/visualization_utils.hpp (hunk 1 above)
Comment (SteveMacenski) [reply include/slam_toolbox/visualization_utils.hpp:49]: This branch has been added so you can retarget it

Comment (SyllogismRXS) [reply include/slam_toolbox/visualization_utils.hpp:49]: Thanks. The target branch is now ros2.

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
Thanks for the contribution!

**Pull request merged**